serve.go

```diff
@@ -507,6 +507,9 @@ func (c *config) depClient() (dep.Client, error) {
 		if err != nil {
 			return err
 		}
+		if !depsim {
+			conf = boltConf
+		}
```

Comment:
Instead of doing this check, assign every value to `boltConf` below, and then set `conf = boltConf`. if theres nothing in the bucket we would've bailed already.